code/datums/view.dm

```diff
@@ -4,6 +4,7 @@
 	var/width = 0
 	var/height = 0
 	var/default = ""
+	var/zoom = 0
```

Comment:
auto-doc please, milord

Reply:
s my shitcode, I'll fix it as I like (sure)